core/services/pipeline/orm.go

```diff
@@ -133,43 +132,17 @@ func (o *orm) CreateRun(ctx context.Context, jobID int32, meta map[string]interf
 			})
 		}
 		runID = run.ID
-		return tx.Create(&trs).Error
+		if len(trs) > 0 {
```

Comment:
Before this we always had at least the result task, so it never errored saying "empty slice"

Reply:
I am happy our tests caught this.